VirtoCommerce.MarketingModule.Web/Scripts/promotion/blades/promotion-expressions.js

```diff
@@ -49,6 +49,7 @@ angular.module('virtoCommerce.marketingModule')
                     parentElement.productId = selectedListEntries[0].id;
                     parentElement.productName = selectedListEntries[0].name;
                     parentElement.productCode = selectedListEntries[0].code;
+                    blade.parentBlade.$scope.$$childTail.productCodes[parentElement.productId] = parentElement.productCode;
```

Comment:
Do we really need to use $$childTail here?

Reply:
Same, what on the first case.